spec/unit/provider/package/rpm_spec.rb

```diff
@@ -7,12 +7,13 @@
 
   let (:packages) do
     <<-RPM_OUTPUT
-    cracklib-dicts 0 2.8.9 3.3 x86_64
+    'cracklib-dicts 0 2.8.9 3.3 x86_64
```

Comment:
The final quote on a line by itself is expected because of the trailing new-line (\\n) in the rpm query formatting string NEVRA_FORMAT.